packages/datepicker/src/__tests__/FzDatepicker.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { describe, it, vi } from "vitest";
+import { mount } from "@vue/test-utils";
+import { FzDatepicker } from "..";
+
+describe.concurrent("FzDatepicker", () => {
+  vi.mock('@fiscozen/composables', () => ({
+    useBreakpoints: vi.fn().mockReturnValue({
+      isSmaller: vi.fn().mockReturnValue(false)
+    })
+  }))
+  it("matches snaphost", async ({ expect }) => {
+    const wrapper = mount(FzDatepicker, {
+      props: {
+        modelValue: new Date()
+      },
+      slots: {},
+    });
+
+    expect(wrapper.html()).toMatchSnapshot();
+  });
```

Comment:
I saw there's a bunch of different stories, you could do a snapshot test for each